plan/plan.go

```diff
@@ -76,6 +79,9 @@ func (p *Plan) Calculate() *Plan {
 		}
 	}
 
+	// Apply policy to list of changes.
+	changes = *p.Policy.Apply(&changes)
```

Comment:
promoting changes by giving it methods is too big of a change for me at this moment. However, I simplified the annoying code a little by storing a pointer to changes.